trade-finance-manager-ui/templates/case/documents/documents.njk

```diff
@@ -2,6 +2,10 @@
 
 {% extends "case/case.njk" %}
 
+{% block pageTitle -%}
+  Deal {{ deal.details.ukefDealId }} - Documents
```

Comment:
Same as above?

Reply:
Yes, it works for GEF, other deals, and all Facilities.